hoptimator-k8s/src/main/java/com/linkedin/hoptimator/k8s/K8sMaterializedViewDeployer.java

```diff
@@ -61,6 +74,15 @@ public void update() throws SQLException {
       deployers.add(pipelineDeployer);
       V1OwnerReference pipelineRef = pipelineDeployer.updateAndReference();
       K8sContext pipelineContext = viewContext.withLabel("pipeline", name).withOwner(pipelineRef);
+      // Only update a TableTrigger if there is exactly one source table and if the OFFLINE_TABLE_NAME hint is set.
+      String offlineTableName = hints.get(OFFLINE_TABLE_NAME);
+      if (view.pipeline().sources().size() == 1 && !Strings.isNullOrEmpty(offlineTableName)) {
```

Comment:
I guess eventually hints will work for multiple sources; e.g.:

```
create materialized view foo as
   select * from bar /*+ hint('offline.table.name'='BAR') */,
   qux /*+ hint('offline.table.name'='QUX') */
```

Reply:
Yeah, it will take a lot of work to get these hints working within our DDL, but it is possible. We've looked into it multiple times, but haven't prioritized it yet. In the meantime, we can enable hints on the entire connection.